src/components/HostStatusBadge.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { HostStatus } from "types/host";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import styled from "@emotion/styled";
+
+interface Props {
+  status: HostStatus;
+}
+
+export const HostStatusBadge: React.FC<Props> = ({ status }) => (
+  <StyledBadge variant={statusToBadgeVariant[status]}>
+    {hostStatusToCopy[status]}
+  </StyledBadge>
+);
+
+const statusToBadgeVariant = {
+  [HostStatus.Running]: Variant.Green,
+  [HostStatus.Terminated]: Variant.Red,
+  [(HostStatus.Starting, HostStatus.Provisioning)]: Variant.Yellow,
+  [(HostStatus.Decommissioned,
+  HostStatus.Quarantined,
+  HostStatus.ProvisionFailed,
+  HostStatus.Uninitialized,
+  HostStatus.Building,
+  HostStatus.StatusSuccess,
+  HostStatus.Stopping,
+  HostStatus.Stopped,
+  HostStatus.StatusFailed,
+  HostStatus.ExternalUserName)]: Variant.LightGray,
```

Comment:
Thats interesting first time ive seen mapping multiple keys within an array to single items as values.

Reply:
Are you sure this works? I tried this out on a smaller scale and it seems like only the last key in the array gets mapped. 
![image](https://user-images.githubusercontent.com/4605522/89221115-b9fbec00-d5a0-11ea-97db-5930c5e31178.png)